### PR TITLE
Delete JD.com-mixedcontent.xml & Update JD.com.xml

### PR DESCRIPTION
**Pull request by renyouguo**
They are ok now.

### CODE REVIEW

**Comment by renyouguo** [comment]
@J0WI Oh, it is so hard to pass the check.

**Comment by renyouguo** [comment]
@J0WI This is one of the most popular site for chinese.
Please check it.

File: src/chrome/content/rules/JD.com.xml [hunk 1]
```diff
@@ -185,106 +97,239 @@
...
 -->
+
 <ruleset name="JD.com (partial)">
 
-	<target host="jd.com" />
-	<target host="apismart.jd.com" />
-	<!--target host="app.jd.com" /-->
+	<!--	Complication	-->
+	
+	<target host="*.jd.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/JD.com.xml]: This will also redirect nonfunctional hosts.

File: src/chrome/content/rules/JD.com.xml [hunk 2]
```diff
@@ -295,36 +340,26 @@
...
-	<!--securecookie host="^zhaopin\.jd\.com$" name="^(?:JSESSIONID|ZHAOPIN_COOKIE)" /-->
-
-	<securecookie host="^\." name="^_gat?$" />
-	<securecookie host="^\w" name="." />
-	<securecookie host="^\.search\.jd\.com$" name="." />
-
-
-	<rule from="^http:"
-		to="https:" />
+	<rule from="^http:" to="https:" />
```
Comment (J0WI) [inline src/chrome/content/rules/JD.com.xml]: This will overwrite all complicate redirects above.

Comment (renyouguo) [inline src/chrome/content/rules/JD.com.xml]: @J0WI add jd.com-problematic.xml ?

Comment (J0WI) [inline src/chrome/content/rules/JD.com.xml]: You can do this or writing exclusions/be more strict in what gets redirected.

**Comment by renyouguo** [comment]
#5626

**Pull request closed without merging**